### DISCUSSION TITLE
V7 - Image / Uploads in repeatable and relationship type

### DISCUSSION THREAD

**Discussion by manfield** [Laravel-Backpack/community-forum#1202] [Q&A (Help)]
Hi, I'm aware about the next milestones and the rework you're doing on upload components.
I wonder if in release v.7 these things will be all fixed:

- full support for Spatie/Medialibrary wherever the image/file field is used  (relatiionship, repeating, etc... subfields)
- all issues fixed for uploaders in every situation (fields, subfields, for all types supported)
- issues with validation for images

Would be extremely helpful. In this moment I have a project developed with backpck that is not consistent in all his parts because of this.
Thanks a lot!

**Comment by karandatwani92** [comment]
Hey @manfield 

The Uploader PR is ready on our side to fix all the above. We have tried our best to keep it BUG-free.
We are now cooking v7 to push this and other features/fixes. We are trying hard to make it with minimum upgrade steps and include everything that could be a breaking change.

Thanks for being patient and trusting us.🤝
